### PR TITLE
Accept lists in Path

### PR DESCRIPTION
**Pull request by basnijholt**
The following code now works:

```
import holoviews as hv
hv.notebook_extension()

x = range(10)
y = range(10)
hv.Curve((x, y)) + hv.Path((x, y)) + hv.Contours((x, y))
```

### CODE REVIEW

**Comment by philippjfr** [comment]
Looks good, will merge when tests pass.

**Comment by philippjfr** [comment]
Will investigate why tests are failing.

File: holoviews/element/path.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class Path(Element2D):
 
     def __init__(self, data, **params):
         if isinstance(data, tuple):
-            x, y = data
+            x, y = map(np.asarray, *data)
```
Comment (philippjfr) [inline holoviews/element/path.py]: This should just be ``map(np.asarray, data)``, could you fix it, then I'll merge asap.

**Comment by basnijholt** [comment]
Thanks, done :)

**Comment by philippjfr** [comment]
Thank you! Ready to merge.

**Pull request merged**